### ISSUE TITLE
Prio.Max.keys, elems: documented order of keys is wrong

### ISSUE THREAD

**Issue by amigalemming** [lspitzner/pqueue#11]
The documentation of `PQueue.Prio.Max.keys` claims:
"Return all keys of the queue in ascending order."

But this is not true:
~~~~
Prelude PQ> PQ.keys $ PQ.fromList [(0,'a'), (5,'b'), (10::Int,'c')]
[10,5,0]
~~~~
However, I think that it is most natural to return the keys in descending order for `MaxPQueue`.
Thus I propose to change the documentation from "ascending" to "descending".

**Comment by lspitzner** [comment]
right. same for `elems`.

**Comment by lspitzner** [comment]
Fixed in 53ac51cb33873b5d452e997809135ccbec4b89e1
